### DISCUSSION TITLE
Can archive.org dynamic theme rules/styles be simplified?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [darkreader/darkreader#6831] [Ideas]
I no longer see `.search-toolbar-logo` on `web.archive.org`.  Could that dynamic theme tweak be removed?  Or is that element on some page that I didn't find?

If the `web.archive.org` devs changed that element, it's possible that other tweaks for that host/domain need to be added/removed/edited.

**Comment by Gitoffthelawn** [comment]
BTW, should this be an issue or is discussions the best place for it?  If an issue, which type of issue?

**Reply by Gusted** [reply]
Let's make a MEGAThread whereby users can report sites which could be simplified in terms of CSS Fixes.

**Reply by Gitoffthelawn** [reply]
Done: https://github.com/darkreader/darkreader/issues/6834

**Comment by Gusted** [comment]
> Or is that element on some page that I didn't find?

One thing is that most element you won't find 1 2 3... and could be on some obscure site, or being a login-wall or pay-wall. You can use the blame tool of git(hub). And look at the relevant PR which has added the fix to find maybe some context about it.

**Reply by Gitoffthelawn** [reply]
Using blame to find the relevant PR is a great idea.

I recently created a custom style for the shadow DOM on that particular site and was using that class selector too.  I didn't go too deep on that site, so I'm guessing it was towards the top level of it... but I can't make any guarantees, as it was several months ago. ;)

I'll use blame to find the pull when I have some time... maybe the author of those fixes will know where that class was used.
